src/processing/dom/base-dom-adapter.js

```diff
@@ -3,7 +3,7 @@
 // Do not use any browser or node-specific API!
 // -------------------------------------------------------------
 
-const IFRAME_FLAG_TAGS = ['a', 'form'];
+const IFRAME_FLAG_TAGS = ['a', 'form', 'area'];
```

Comment:
I believe we can use https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/processing/dom/index.js#L37